<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindoweditinfo</class>
 <widget class="QMainWindow" name="MainWindoweditinfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_ChangeUserInfo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>531</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_Idchange_heading">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>241</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <fontweight>Medium</fontweight>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;
font-size:  50px;
font-weight: 500;</string>
     </property>
     <property name="text">
      <string>Edit Data</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_Email_button">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>250</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font-size: 15px;</string>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="changed_text_for_UniqueId">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>330</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_SaveTheChange">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>390</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: blue;
font-size: larger;
font-weight: 400;
border: none;
border-radius: 20px;
background-color: white;</string>
     </property>
     <property name="text">
      <string>Save Changes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_InfoChange_prompt">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font-size: 15px;</string>
     </property>
     <property name="text">
      <string>Which Info would you like to change:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_Username_button">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>250</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font-size: 15px;</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Enter_UniqueId">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font-size: 15px;</string>
     </property>
     <property name="text">
      <string>Enter the User/ Seller ID: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Unique_id_input">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Enter_theChnagedText">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>300</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font-size: 15px;</string>
     </property>
     <property name="text">
      <string>Enter the Changed Text:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_Password_button">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>250</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font-size: 15px;</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
